README.md

```diff
@@ -0,0 +1,25 @@
+# Zarr Specs
+
+![](https://raw.githubusercontent.com/zarr-developers/community/main/logos/logo2.png)
```

Comment:
FYI: https://github.com/zarr-developers/community/blob/main/logos/README.md

Reply:
Thanks, good point. I copied that link from the zarr-python Readme, I'll add the change there, too.
```suggestion
![](https://github.com/zarr-developers/zarr-logo/releases/download/2022-04-28/zarr-pink-horizontal-transparent.png)
```

Reply:
PS: The `horizontal` pngs contain a lot of whitespace on the top and bottom and look weird, when included in the Readme directly. Therefore I used the stacked version for now.